db/seeds.rb

```diff
@@ -6,24 +6,24 @@
 
 item7 = chow.items.create!(title: 'Artichoke Souffle', description: 'Try this rich-tasting cheese, artichoke and spinach soufflé recipe for your next brunch.', price: 700, image: File.new("#{Rails.root}/app/assets/images/breakfast.jpg"))
 item8 = chow.items.create!(title: 'Berry Tart', description: 'Lightly sweetened berries top vanilla bean-flecked pastry cream contained in a graham cracker crust.', price: 500, image: File.new("#{Rails.root}/app/assets/images/tarts.jpg"))
 item9 = chow.items.create!(title: 'Caprese Panini', description: 'The classic combination of mozzarella, tomatoes and basil is known as caprese.', price: 700, image: File.new("#{Rails.root}/app/assets/images/panini.jpg"))
-item10 = chow.items.create!(title: 'Blueberry Muffin', description: 'blueberry muffins balance a moist, buttery crumb topping and are equally delicious with blueberries in season.', price: 400, image: File.new("#{Rails.root}/app/assets/images/muffin.jpg"))
+item10 = chow.items.create!(title: 'Blueberry Muffin', description: 'Blueberry muffins balance a moist, buttery crumb topping and are equally delicious with blueberries in season.', price: 400, image: File.new("#{Rails.root}/app/assets/images/muffin.jpg"))
 item11 = chow.items.create!(title: 'Wedge Salad', description: 'Hearts of romaine lettuce, painted with a herb vinaigrette, and grilled.', price: 800, image: File.new("#{Rails.root}/app/assets/images/salad.jpg"))
 item12 = chow.items.create!(title: 'Tomato Soup and Toasted Ravioli', description: 'Creamy tomatoe soup made from freash local ingrediants, garnished with our in house toasted ravioli pasta.', price: 800, image: File.new("#{Rails.root}/app/assets/images/soup.jpg"))
 
 item13 = coffee.items.create!(title: 'Capuccino', description: 'Our Creamy cappuccino offers a stronger espresso flavor and a luxurious texture.', price: 500, image: File.new("#{Rails.root}/app/assets/images/biscotti.jpg"))
 item14 = coffee.items.create!(title: 'Chocolate Capuccino', description: 'Chocolaty sweetness and hearty coffee unite to create a decadent cup of cappuccino.', price: 500, image: File.new("#{Rails.root}/app/assets/images/choc-coffee.jpg"))
 item15 = coffee.items.create!(title: 'Cinnful Capuccino', description: 'An espresso and cinnamon-scented scone with cappuccino.', price: 500, image: File.new("#{Rails.root}/app/assets/images/cinn-tea.png"))
-item16 = coffee.items.create!(title: 'Cocoa', description: 'delicious organic chocolate to be savoured as it is or combined with intriguing infusions.', price: 500, image: File.new("#{Rails.root}/app/assets/images/coco.jpg"))
+item16 = coffee.items.create!(title: 'Cocoa', description: 'Delicious organic chocolate to be savoured as it is or combined with intriguing infusions.', price: 500, image: File.new("#{Rails.root}/app/assets/images/coco.jpg"))
```

Comment:
Useless assignment to variable - `item16`.<br>Line is too long. [225/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.